eslint.config.mjs

```
@@ -15,7 +15,7 @@ export default [
   ...nx.configs['flat/typescript'],
   ...nx.configs['flat/javascript'],
   {
-    ignores: ['**/dist', '**/out-tsc', '**/vitest.config.*.timestamp*', '**/.vitepress/cache', '**/.vitepress/dist', '**/vite.config.*.timestamp*'],
+    ignores: ['**/dist', '**/out-tsc', '**/vitest.config.*.timestamp*', '**/.vitepress/cache', '**/.vitepress/dist', '**/vite.config.*.timestamp*', 'apps/docs/public/demos/**'],
```

ESLint

[error] Replace `'**/dist',·'**/out-tsc',·'**/vitest.config.*.timestamp*',·'**/.vitepress/cache',·'**/.vitepress/dist',·'**/vite.config.*.timestamp*',·'apps/docs/public/demos/**'` with `⏎······'**/dist',⏎······'**/out-tsc',⏎······'**/vitest.config.*.timestamp*',⏎······'**/.vitepress/cache',⏎······'**/.vitepress/dist',⏎······'**/vite.config.*.timestamp*',⏎······'apps/docs/public/demos/**',⏎····`

(prettier/prettier)